cmd/brew-setup-nginx-conf

```diff
@@ -17,7 +17,7 @@ root = ARGV.shift || "."
 input = ARGV.shift || "config/dev/nginx.conf.erb"
 
 if !name || !root || !input
-  abort "Usage: brew generate-nginx-conf [--root] <project_name> <project_root_path> <nginx.conf.erb>"
+  abort "Usage: brew generate-nginx-conf [--root] <project_name> [<project_root_path> [<nginx.conf.erb>]]"
```

Comment:
`<project_name> [project_root_path [nginx.conf.erb]]` would be more consistent with our typical format. Thanks though 👍